bot/utility/utility.py

```diff
@@ -95,6 +95,47 @@ async def about(self, ctx: commands.Context):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.group(name='howto', invoke_without_command=True)
+    @command_log
+    async def howto(self, ctx: commands.Context, subcommand: Optional[str]):
+        """Handler for the `howto` command.
+
+        Explains how to do various things
```

Comment:
Missing dot at the end of the sentence but I would generally change it to something likes this:
> The available subcommands of this Command Group post individual help messages explaining how to do various things regarding the Discord server itself or other often needed tasks.